### PR TITLE
Create LIT6831PrayerGeorge

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created an ID for LIT6831PrayerGeorge as attested in BMLacq306. A very short prayer, that is very similar to the final part of LIT4518Hymn. Although very short altogether, I decided to create a new ID for this piece as discussed in issue #2319. I gave the entire text as a div.

### CODE REVIEW

File: new/LIT6831PrayerGeorge.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Very short prayer to George, that is similar to the closing part of <title ref="LIT4518Hymn"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Prayers"/>
```
Comment (thea-m) [inline new/LIT6831PrayerGeorge.xml:39]: also ChristianLiterature

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, just minor technical remarks

File: new/LIT6831PrayerGeorge.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                </keywords>
+            </textClass>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/LIT6831PrayerGeorge.xml:42]: Done.

File: new/LIT6831PrayerGeorge.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6831PrayerGeorge.xml:39]: Done.

**Pull request merged**